.github/workflows/build_and_test.yaml

```diff
@@ -43,27 +45,27 @@ jobs:
           - maj_min: v1.22
             digest: sha256:f5b2e5698c6c9d6d0adc419c0deae21a425c07d81bbf3b6a6834042f25d4fba2
```

Comment:
kind 0.22.0 doesn't seem to officially support anything below 1.23 anymore. Shouldn't we delete these then?

Reply:
Thx for pointing out. I agree to remove v1.21 and v1.22